indico/modules/rb_new/client/js/modules/bookRoom/selectors.js

```diff
@@ -23,6 +23,6 @@ export const isFetchingFormTimeline = ({bookRoom}) => {
     return bookRoom.bookingForm.requests.timeline.state === RequestState.STARTED;
 };
 export const getUnavailableRoomInfo = ({bookRoom: {unavailableRooms}}) => unavailableRooms.availability;
-export const getTimelineDateRange = ({bookRoom: {timeline}}) => timeline.dateRange;
+export const getUnavailableTimelineDateRange = ({bookRoom: {unavailableRooms}}) => unavailableRooms.dateRange;
```

Comment:
this change doesn't seem related to your fix; can we skip this (and the change in UnavailableRoomsModal) since it will already be changed in my PR (to take it from the meta data i return together with the search results - the "feels slightly dirty" one I mentioned in the meeting today)

The range for unavailable rooms is always the same as the range we searched for initially, right?